ssl/ssl_lib.c

```diff
@@ -4413,15 +4413,20 @@ int ssl_log_rsa_client_key_exchange(SSL *ssl,
                                     const uint8_t *premaster,
                                     size_t premaster_len)
 {
+    uint8_t premaster_tag[8];
```

Comment:
It seems that you don't use this for anything useful.  Isn't it enough that the length parameter for the encrypted premaster was changed to 8 in the call to `nss_keylog_int`?